code/modules/clothing/head/helmet.dm

```diff
@@ -82,11 +89,14 @@
 	name = "handmade combat helmet"
 	desc = "It looks like it was made from a bucket and some steel. Uncomfortable and heavy but better than nothing."
 	icon_state = "hm_helmet"
-	armor = list(melee = 35, bullet = 30, energy = 30, bomb = 0, bio = 0, rad = 0)
-	flags_inv = HIDEEARS
```

Comment:
Do note these changes remove the HIDEEARS flags on helmets and none of the new helmets have that flag either.

Reply:
/obj/item/clothing/head/helmet is already set to HIDEEARS so it isn't required to have that on /obj/item/clothing/head/helmet/handmade. ; )